linode/instance/resource.go

```diff
@@ -725,3 +799,9 @@ func deleteResource(ctx context.Context, d *schema.ResourceData, meta interface{
 	d.SetId("")
 	return nil
 }
+
+func populateLogAttributes(ctx context.Context, d *schema.ResourceData) context.Context {
+	return helper.SetLogFieldBulk(ctx, map[string]any{
+		"lid": d.Id(),
```

Comment:
Maybe calling it `linode_id` would be better?
```suggestion
		"linode_id": d.Id(),
```